examples/test.yaml

```diff
@@ -0,0 +1,23 @@
+type: prompt-engine
+config:
+  model_config:
+    max_tokens: 1024
+  description_prefix: "Hello"
```

Comment:
I'm not sure why someone would ever have description_prefix as "Hello", versus just adding that in the description. For a chat example with yaml, I'd sooner leave description_prefix empty, since it doesn't really make sense to have one and may confuse folks

Reply:
I'll change the example config to a bit to make it better